impeller/renderer/backend/vulkan/allocator_vk.cc

```diff
@@ -196,9 +221,6 @@ static VmaAllocationCreateFlags ToVmaAllocationCreateFlags(StorageMode mode,
       }
       return flags;
     case StorageMode::kDevicePrivate:
-      if (is_texture) {
```

Comment:
Lets not use VMA_ALLOCATION_CREATE_DEDICATED_MEMORY_BIT until we have a case where it performs better

Reply:
I thought Angle did that for > 4MB allocations though. Perhaps just start off with that assumption, but for non-render targets?

Reply:
Lets follow up on this one in https://github.com/flutter/engine/pull/43325 ?

If we don't remove this bit we'll have the same performance issues as dedicated allocations don't go into the pool.

Reply:
Sounds good.